contracts/token/LivepeerToken.sol

```diff
@@ -1,12 +1,55 @@
-pragma solidity ^0.5.11;
+//SPDX-License-Identifier: MIT
+// solhint-disable-next-line
+pragma solidity ^0.8.0;
```

Comment:
livepeer/arbitrum-lpt-bridge#54 changes solidity version of LPT to 0.8.8. Should use the same version here as well.

Reply:
Fixed in 4f8b02d